plugins/jdbc-adapter-framework/build.gradle

```diff
@@ -6,6 +6,8 @@ dependencies {
     compileOnly project(":plugins:sql-language")
     api group: 'org.apache.commons', name: 'commons-dbcp2', version: commons_dbcp2_version
 
+    implementation group: "net.postgis", name: "postgis-jdbc", version: postgis_version
```

Comment:
Why is this imported here and not in the postgresql plugin?

Reply:
Moved to postgres plugin